src/app/stripe.service.ts

```diff
@@ -241,3 +241,53 @@ export class StripeService {
     return Math.round((donation.tipAmount + donation.donationAmount) * 100);
   }
 }
+
+
+export function getStripeFriendlyError(
+  error: StripeError | {message: string, code: string, decline_code?: string, description?: string} | undefined,
+  context: 'method_setup'| 'card_change'| 'confirm',
```

Comment:
Liking this narrower type; assuming rest of this is moved only

Reply:
Yes it is just moved.